### PR TITLE
Update contact.md

### PR DESCRIPTION
**Pull request by su-chang**
1. 明确API类型；
2. 修复部分Bug；

### CODE REVIEW

File: root/api/contact.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ ContactType 在这里是enum
 ```javascript
 const bot = new Wechaty()
 await bot.start()
-const isOfficial = contact.type() === bot.Contact.Type.Official
+const isOfficial = contact.type() === bot.ContactType.Official
```
Comment (lijiarui) [inline root/api/contact.md]: Why?

File: root/api/contact.md [hunk 2]
```diff
@@ -183,7 +183,7 @@ ContactGender在这里是 enum
 **Example**
 
 ```javascript
-const gender = contact.gender() === bot.Contact.Gender.Male
+const gender = contact.gender() === bot.ContactGender.Male
```
Comment (lijiarui) [inline root/api/contact.md]: Why?

File: root/api/contact.md (hunk 1 above)
Comment (su-chang) [reply root/api/contact.md]: Sorry, it's my fault. I misunderstand the type between wechaty and puppet. I will correct right now.

File: root/api/contact.md (hunk 2 above)
Comment (su-chang) [reply root/api/contact.md]: Done. I misunderstand the type between wechaty and puppet. I add this to the rule about writing API of wechaty.

**Pull request merged**